src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/CreateCaseTest.java

```diff
@@ -88,10 +92,30 @@ public void should_create_case_from_valid_exception_record() throws Exception {
         assertThat(createdCase.getData().get("email")).isEqualTo("hello@test.com");
     }
 
+    @Test
+    public void should_clear_exception_record_warnings() throws Exception {
```

Comment:
The story mentions functional test, but having written it, I'm not entirely sure it's needed - we have it pretty much covered in the integration test and there's not much more we can check here.